test/infra/Makefile

```diff
@@ -2,14 +2,13 @@
 # ---------------------------------------------------------------------
 export GOOS ?= $(shell go env GOOS 2>/dev/null)
 export GOARCH ?= $(shell go env GOARCH 2>/dev/null)
-export TERRAFORM_VERSION ?= 0.15.3
+export TERRAFORM_VERSION ?= 1.5.2
```

Comment:
Does bumping this have side effects for any existing automation?

Reply:
Good point. let me run test on all the providers.